lib/kitchen/directions/bake_iframes/v1.rb

```diff
@@ -25,7 +20,7 @@ def bake(book:)
 
         iframe.prepend(child:
           <<~HTML
-            <a class="os-is-link" href="#{iframe_link}" target="_blank" rel="noopener nofollow">#{I18n.t(:iframe_link_text)}</a>
+            <a class="os-is-link" data-needs-rex-link="true" href="#{iframe_link}" target="_blank" rel="noopener nofollow">#{I18n.t(:iframe_link_text)}</a>
```

Comment:
So postbake uses this data attribute to do something? I tried looking it up but didn't find anything (maybe it's not found with the full attribute name?)

Reply:
It's in the other PR: openstax/enki#405.

Reply:
Ah thanks I did not notice the second link